libvirt/resource_libvirt_network_test.go

```diff
@@ -130,3 +133,96 @@ func TestAccLibvirtNetwork_Import(t *testing.T) {
 		},
 	})
 }
+
+func TestAccLibvirtNetwork_EnableDhcpAndDisableItAfter(t *testing.T) {
```

Comment:
rename this

Reply:
To whom are you speaking to? :smile_cat:

Reply:
i just reviewed my code myself :rocket: :robot: was a sidenote :rofl:

Reply:
Please don't get angry with your reviewer :yum: . Enjoy your week-end too :fireworks: .